projects/control-service/projects/helm_charts/pipelines-control-service/templates/deployment.yaml

```diff
@@ -242,10 +244,13 @@ spec:
             - name: {{ $key }}
               value: "{{ $value }}"
             {{- end }}
+
           volumeMounts:
             - name: secrets
               mountPath: "/etc/secrets"
               readOnly: true
+            - mountPath: "/DATAJOBS_TEMP_STORAGE_FOLDER"
```

Comment:
> made a release of the helm chart under a different name and deployed to the vdk vdp cluster.
> Deployed a heartbeat job.

could you also paste the test result file structure found?

Reply:
I can't actually provide you with this because the files are cleaned up after each operation.
If you really want to see if I could try run the `ls` operations while it is in the middle of an upload. 
what do you want to see it?

Reply:
here is the result. 
```
I have no name!@vdk-control-service-dep-7ffd6f875c-5pnvd:/application$ ls /DATAJOBS_TEMP_STORAGE_FOLDER
job_562d678e-0396-4aa3-a924-418701ebf2c1
```